lib/feature_management.rb

```diff
@@ -81,8 +81,8 @@ def self.current_env_allowed_to_see_usps_code?
     ENVS_WHERE_PREFILLING_USPS_CODE_ALLOWED.include?(Figaro.env.domain_name)
   end
 
-  def self.no_pii_mode?
-    enable_identity_verification? && Figaro.env.profile_proofing_vendor == :mock
+  def self.fake_banner_mode?
+    Rails.env.production? == false && Figaro.env.domain_name != 'secure.login.gov'
```

Comment:
Thanks for the updates. This looks great. I failed to clarify this earlier: what we mean by "the banner should show up on all servers except for production" is that the logic is based on the server host, which is defined in `Figaro.env.domain_name`. The Rails environment is `production` on all of those servers. When I suggested we use `Rails.env.production?` as the first condition, I meant the banner should not show up when developing the app locally and in the test environment.

So, what we want is `Rails.env.production?`, not `Rails.env.production? == false`.

Reply:
Done